xml/article_sap_trento.xml

```diff
@@ -293,7 +293,17 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </para>
       </listitem>
     </itemizedlist>
-    <para> To install Trento on your &t.server; host proceed as follows: </para>
+
+    <para>
+      The &t.server; is available as package <package>trento-premium</package> in your
+      respective &suse; repository. The package does not require &productname;.
+      This package is available for customers that do not have a K3s; in place and
+      want to deploy the server on K3s for &productname;.
+    </para>
+
+    <para>
+      If you cannot or do not want to install the <package>trento-premium</package>
+      package, install Trento on your &t.server; host as follows: </para>
```

Comment:
I would say something like: The trento-premium-server-installer simplifies the installation and installs K3s and helm for you, so it replaces the following manual steps.